definitions/npm/kafkajs_v2.x.x/test_kafkajs_v2.x.x.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import { describe, it } from "flow-typed-test";
+import { Kafka, Producer, Consumer, Partitioners, logLevel } from "kafkajs";
+
+describe("kafkajs", () => {
+  it("creates Kafka instance", () => {
+    const kafka = new Kafka({
+      clientId: "my-app",
+      brokers: ["kafka1:9092", "kafka2:9092"],
+    });
+  });
+
+  it("creates Consumer instance", () => {
+    const kafka = new Kafka({
+      clientId: "my-app",
+      brokers: ["kafka1:9092", "kafka2:9092"],
```

Comment:
Suggestions for the constructor, you might want to exercise the options,

- can it accept all the expected options
- what if you pass in the invalid values for the option
- what if you pass in an option that doesn't exist?

Then once constructed with `kafka` returned to you, does it has the functions and properties you expect but at the same time, does it do things you don't expect?